### PR TITLE
Fix output of log: date is one day off

### LINKED ISSUE #18632: incorrect output
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Text_formatting

### What specific section or headline is this issue about?

date and time formating

### What information was incorrect, unhelpful, or incomplete?

this is the exemple in date and time formating section

```js
const msPerDay = 24 * 60 * 60 * 1000;

// July 17, 2014 00:00:00 UTC.
const july172014 = new Date(msPerDay * (44 * 365 + 11 + 197));

const options = { year: '2-digit', month: '2-digit', day: '2-digit',
                hour: '2-digit', minute: '2-digit', timeZoneName: 'short' };
const americanDateTime = new Intl.DateTimeFormat('en-US', options).format;

console.log(americanDateTime(july172014)); // 07/16/14, 5:00 PM PDT
```

17 july 2014 should be 07/17/14

### What did you expect to see?

07/17/14, 7:00 AM GMT+7

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/guide/text_formatting`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Text_formatting
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/guide/text_formatting/index.md
* Last commit: https://github.com/mdn/content/commit/426485595b1be50e2af91764fbfd94c04c3c36a5
* Document last modified: 2022-02-18T11:38:26.000Z

</details>

### PR DESCRIPTION
**Pull request by teoli2003**
Fixes #18632

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
The original text is correct if you are in PDT—`5:00 PM PDT` and `2:00 AM GMT+2` are the same time! We should use `Z` for the original date creation, or at least use UTC time in the example output, combined with your note about timezone awareness.

File: files/en-us/web/javascript/guide/text_formatting/index.md [hunk 1]
```diff
@@ -272,7 +272,10 @@ const options = { year: '2-digit', month: '2-digit', day: '2-digit',
                 hour: '2-digit', minute: '2-digit', timeZoneName: 'short' };
 const americanDateTime = new Intl.DateTimeFormat('en-US', options).format;
 
-console.log(americanDateTime(july172014)); // 07/16/14, 5:00 PM PDT
+// Local timezone vary depending on your settings
+// In CEST, logs: 07/17/14, 02:00 AM GMT+2
+// In PDT, logs: 07/16/14, 05:00 PM PDT
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/text_formatting/index.md:277]:
```suggestion
// In PDT, logs: 07/16/14, 05:00 PM GMT-7
```

**Review by Josh-Cena — APPROVED** [review]
If I'm to write this code example, I wouldn't be creating the date from a timestamp at all—seems pretty weird to demonstrate formatting this way. But I think it's fine in its current form. Do you think changing to `new Date("2014-07-17")` is better?

**Comment by teoli2003** [comment]
Definitively. Fixed.

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**